metadata_backend/api/operators.py

```diff
@@ -138,6 +150,26 @@ async def _remove_object_from_db(self,
             LOG.error(reason)
             raise web.HTTPBadRequest(reason=reason)
 
+    async def _insert_new_folder_to_db(self, data: Dict) -> str:
+        """Insert new object folder to database.
+
+        :param data: JSON object containing initial folder info
+        :returns: Folder ID for folder inserted to database
+        """
+        try:
+            data['metadata_objects'] = []
```

Comment:
I don't think there's need to add empty list? User can see there's no objects, if there's only name and description available